src/main/java/com/opentable/kafka/embedded/EmbeddedKafkaBroker.java

```diff
@@ -117,6 +117,7 @@ private KafkaConfig createConfig()
         config.put(KafkaConfig.DeleteTopicEnableProp(), "false");
         config.put(KafkaConfig.ControlledShutdownRetryBackoffMsProp(), "100");
         config.put(KafkaConfig.LogCleanerDedupeBufferSizeProp(), "2097152");
+        config.put(KafkaConfig.OffsetsTopicReplicationFactorProp(), (short) 1);
```

Comment:
Default values are set in the broker configuration, and apps can override them if need be, but I don't think there's a global "right" value.